red-hat-storage/ocs-ci
# verify coredump getting generated for ceph mon daemon crash

Test Procedure:
1.login to OCS node
$ oc debug node/compute-0

2.Kill ceph-mon process
sh-4.4# ps -aef|grep ceph-mon  
167      1844505 1844493  0 Feb08 ?        00:19:47 ceph-mon --fsid=1413b
sh-4.4# kill -11 1844505

3.Connect to ceph tools pod and verify that we have "ceph-mon crash"
$ oc rsh rook-ceph-tools-84bc476959-2d2ps
sh-4.4# ceph crash ls
ID                                                               ENTITY NEW 
2021-02-10_10:21:02.360184Z_93f26ef2-bdee-407f-9581-cb5d0faa931c mon.b   *  

4.Check for crash log and coredump
$ oc debug node/compute-0
sh-4.2# chroot /host
sh-4.4# coredumpctl list
TIME                            PID   UID   GID SIG COREFILE  EXE
Tue 2021-02-09 06:23:16 UTC  3126591 1000620000     0  11 present   /usr/libexec/platform-python3.6
Wed 2021-02-10 10:21:23 UTC  1844505   167   167  11 present   /usr/bin/ceph-mon


Teardown:
ceph crash archive-all

bz: https://bugzilla.redhat.com/show_bug.cgi?id=1904917
Signed-off-by: Oded Viner <oviner@redhat.com>
